src/pages/integrations/unreal.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { LandingPage } from '../../component/landingPages/LandingPage';
+import { LandingPageHeadline } from '../../component/landingPages/LandingPageHeadline';
+import { LandingPageDescription } from '../../component/landingPages/LandingPageDescription';
+import { LandingPageInstallationStep } from '../../component/landingPages/LandingPageInstallationStep';
+import { LandingPageInstallation } from '../../component/landingPages/LandingPageInstallation';
+import UnrealLogo from '../../../static/img/technologies/logo-unreal-engine.svg';
+import { LandingPageActions } from '../../component/landingPages/LandingPageActions';
+import { MoreIntegrations } from '../../component/landingPages/LandingPageMoreIntegrations';
+import Head from '@docusaurus/Head';
+
+const DocsLinks = ({ primary }: { primary?: boolean }) => (
+  <LandingPageActions
+    docs={{ link: '/platform/integrations/unreal_plugin/usage' }}
+    additionalDocs={{
+      label: 'Unreal Marketplace',
+      link: 'TODO',
```

Comment:
I totally agree. I was a bit hesitant to make the change because I've seen the `additionalDocs` didn't have an `icon` field. 

It's alright if I will be adding one for you?

Reply:
I found these cool "Unreal Marketplace icons", let me know how you feel about them:

![image](https://github.com/tolgee/documentation/assets/21221169/449f3f75-40fe-4618-a88c-d69f732f5821)
![image](https://github.com/tolgee/documentation/assets/21221169/f2798afe-ceec-48f9-b943-ef5ec08b08a8)

Reply:
Screenshots have been updated, unfortunately, I didn't get the prettier to work locally, but I will be making the adjustments based on the GitHub comments and fixing them up one by one.

Let me know how you feel about the custom icon implementation

Reply:
As for the Marketplace icon, I am not sure about it because it uses the old Unreal logo inside (see [their brand guidelines](https://epicgames.ent.box.com/s/c2m8idcyejqvg5mjf4e2q73b7jbaghft/file/929793826762)). And as there is no official logo for their Marketplace, I am okay with using the Unreal logo there instead 🙂

Ah, thank you, the screenshots look much better! Could you please fix [this last one issue](https://github.com/tolgee/documentation/pull/420/commits/8f3060a142c53d9042883026a20196dead3e21f7#r1219562570)?

Thanks for the custom icon implementation, I have reviewed it and left only a small comment :)

Reply:
My pleasure. I've added the sizes from the issue and replaced the marketplace logo with the default Unreal Engine logo